src/test/silenceErrors.ts

```diff
@@ -4,6 +4,7 @@ const originalConsoleError = console.error;
 const ignoreErrorsList = [
   `Could not parse CSS stylesheet`,
   `Using kebab-case for css properties in objects is not supported.`,
+  `Warning: Received \`%s\` for a non-boolean attribute \`%s\``, //  See if updating @grafana/ui fixes this one. It is related to the invalid prop on <Field />
```

Comment:
Linking the [issue](https://github.com/grafana/synthetic-monitoring-app/issues/799) so we don't forget to check this once it's upgraded.